test/aensDelegation.js

```diff
@@ -84,27 +84,42 @@ describe('AensDelegation', async () => {
         it('should add and get pointers correctly', async () => {
             // a little bit inconvenient right now, we're discussing how to deal with that in the following issue
             // => https://github.com/aeternity/aepp-sdk-js/issues/1332
-            const accountPointerKey = POINTER_KEY_BY_PREFIX['ak'];
-            const oraclePointerKey = POINTER_KEY_BY_PREFIX['ok'];
-            const contractPointerKey = POINTER_KEY_BY_PREFIX['ct'];
-            const channelPointerKey = POINTER_KEY_BY_PREFIX['ch'];
-            const expectedPointerResult = new Map();
             let getPointersResult = await contract.methods.get_pointers(testName);
-            assert.deepEqual(getPointersResult.decodedResult, expectedPointerResult);
-            const accountPt = { 'AENS.AccountPt': [nameOwnerAddress] };
-            const contractPt = { 'AENS.ContractPt': [nameOwnerAddress] }; // faked contract (can be set!)
-            const oraclePt = { 'AENS.OraclePt': [nameOwnerAddress] }; // faked oracle (can be set!)
-            const channelPt = { 'AENS.ChannelPt': [nameOwnerAddress] }; // faked channel (can be set!)
-            await contract.methods.add_key(nameOwnerAddress, testName, accountPointerKey, accountPt, aensDelegationSignature, { onAccount: delegateAccount });
-            await contract.methods.add_key(nameOwnerAddress, testName, contractPointerKey, contractPt, aensDelegationSignature, { onAccount: delegateAccount });
-            await contract.methods.add_key(nameOwnerAddress, testName, oraclePointerKey, oraclePt, aensDelegationSignature, { onAccount: delegateAccount });
-            await contract.methods.add_key(nameOwnerAddress, testName, channelPointerKey, channelPt, aensDelegationSignature, { onAccount: delegateAccount });
-            expectedPointerResult.set('account_pubkey', accountPt);
-            expectedPointerResult.set('contract_pubkey', contractPt)
-            expectedPointerResult.set('oracle_pubkey', oraclePt);
-            expectedPointerResult.set('channel', channelPt);
+            assert.deepEqual(getPointersResult.decodedResult, new Map());
+
+            // TODO: move to SDK
```

Comment:
@davidyuk do we have an issue for that in the SDK or do we need to create one?

Reply:
we have one: https://github.com/aeternity/aepp-sdk-js/issues/1332